Fix multiple derives

```
 buildpacks-ruby   test    1 error     1 fail   

 1  error[E0252]: the name `_cache_diff` is defined multiple times                                                                                                                                                                                            ▐
   --> commons/src/layer/diff_migrate.rs:211:64                                                                                                                                                                                                               ▐
                                                                                                                                                                                                                                                              ▐
 2  failed: commons/src/layer/diff_migrate.rs - layer::diff_migrate (line 187)                                                                                                                                                                                ▐
    |
 10 | #[derive(Deserialize, Serialize, Debug, Clone, Eq, PartialEq, CacheDiff)]
    |                                                               --------- previous import of the module `_cache_diff` here
 ...
 27 |  #[derive(Deserialize, Serialize, Debug, Clone, Eq, PartialEq, CacheDiff)]
    |                                                                ^^^^^^^^^ `_cache_diff` reimported here
    |
    = note: `_cache_diff` must be defined only once in the type namespace of this block
    = note: this error originates in the derive macro `CacheDiff` (in Nightly builds, run with -Z macro-backtrace for more info)
```

```rust
use magic_migrate::TryMigrate;
use cache_diff::CacheDiff;
use serde::{Deserialize, Serialize};

#[derive(Deserialize, Serialize, Debug, Clone, Eq, PartialEq, CacheDiff)]
#[serde(deny_unknown_fields)]
pub(crate) struct MetadataV1 {
    #[cache_diff(rename = "Ruby version")]
    pub(crate) version: String,
}
 #[derive(Debug)]
pub(crate) enum MigrationError {}
impl std::fmt::Display for MigrationError {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        todo!()
    }
}

#[derive(Deserialize, Serialize, Debug, Clone, Eq, PartialEq, CacheDiff)]
#[serde(deny_unknown_fields)]
pub(crate) struct MetadataV2 {
    #[cache_diff(rename = "Ruby version")]
    pub(crate) version: String,

    #[cache_diff(rename = "OS distribution")]
    pub(crate) distro: String
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Fix: Multiple `#[derive(CachDiff)]` calls in the same file now work (https://github.com/heroku-buildpacks/cache_diff/pull/4)
+
 ## 1.0.0
 
 - Changed: Error when deriving CacheDiff when zero comparison fields are found. This can happen if the struct has no fields or if all fields are `ignore`-d (https://github.com/schneems/cache_diff/pull/4)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,7 +7,7 @@ members = [
 ]
 
 [workspace.package]
-version = "1.0.0"
+version = "1.0.1"
 rust-version = "1.82"
 edition = "2021"
 license = "Apache-2.0"

--- a/cache_diff_derive/src/fields.rs
+++ b/cache_diff_derive/src/fields.rs
@@ -105,9 +105,7 @@ pub fn create_cache_diff(item: TokenStream) -> syn::Result<TokenStream> {
         ))
     } else {
         Ok(quote! {
-            #[allow(clippy::useless_attribute)]
-            use cache_diff as _cache_diff;
-            impl _cache_diff::CacheDiff for #struct_identifier {
+            impl cache_diff::CacheDiff for #struct_identifier {
                 fn diff(&self, old: &Self) -> Vec<String> {
                     let mut differences = Vec::new();
                     #(#comparisons)*

--- a/usage/src/multiple_structs.rs
+++ b/usage/src/multiple_structs.rs
@@ -1,0 +1,34 @@
+// Ensure multiple derives can be in the same file
+#[derive(CacheDiff)]
+struct Dog {
+    woof: String,
+}
+#[allow(dead_code)]
+#[derive(CacheDiff)]
+struct Cat {
+    meow: String,
+}
+#[cfg(test)]
+mod test {
+    use super::*;
+    #[test]
+    fn test_cat() {
+        let diff = Cat {
+            meow: "Meow".to_string(),
+        }
+        .diff(&Cat {
+            meow: "Woem".to_string(),
+        });
+        assert!(diff.len() == 1);
+    }
+    #[test]
+    fn test_dog() {
+        let diff = Dog {
+            woof: "Woof".to_string(),
+        }
+        .diff(&Dog {
+            woof: "Foow".to_string(),
+        });
+        assert!(diff.len() == 1);
+    }
+}

